packages/sdk/src/__tests__/handlePanic.test.ts

```diff
@@ -0,0 +1,135 @@
+import mkdir from 'make-dir'
+import { stdin } from 'mock-stdin'
+import { join, resolve } from 'path'
+import prompt from 'prompts'
+import stripAnsi from 'strip-ansi'
+import tempy from 'tempy'
+
+import { ErrorArea, jestConsoleContext, jestContext, RustPanic } from '..'
+import * as sendPanicUtils from '../sendPanic'
+import * as githubUtils from '../utils/getGithubIssueUrl'
+import { handlePanic } from '../utils/handlePanic'
+
+const keys = {
+  up: '\x1B\x5B\x41',
+  down: '\x1B\x5B\x42',
+  enter: '\x0D',
+  space: '\x20',
+}
+const sendKeystrokes = async (io) => {
+  io.send(keys.down)
+  io.send(keys.enter)
+  await delay(10)
+}
+// helper function for timing
+const delay = (ms) => new Promise((resolve) => setTimeout(resolve, ms)) // Mock stdin so we can send messages to the CLI
+
+const testRootDir = tempy.directory()
+
+// eslint-disable-next-line @typescript-eslint/unbound-method
+const oldProcessCwd = process.cwd
+
+const ctx = jestContext.new().add(jestConsoleContext()).assemble()
+
+describe('handlePanic', () => {
+  // testing with env https://stackoverflow.com/a/48042799/1345244
+  const OLD_ENV = process.env
+
+  beforeEach(async () => {
+    jest.resetModules() // most important - it clears the cache
+    process.env = { ...OLD_ENV } // make a copy
+    process.cwd = () => testRootDir
+    await mkdir(testRootDir)
```

Comment:
Should not be required: https://github.com/sindresorhus/tempy#tempydirectoryoptions

Reply:
This is straight-out copied from the previously existing [handlePanic.test.ts](https://github.com/prisma/prisma/blob/c1d4f6ea4a0499759dcc1e43cc6c846ccb8051a8/packages/migrate/src/__tests__/handlePanic.test.ts#L56-L61) file. I can "modernize" it as you suggest, though

Reply:
Not necessary if it effetively works, on the other hand, boy scout rule 😆